src/Processor/TrimProcessor.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the BibTex Parser.
+ *
+ * (c) Florent DESPIERRES <florent@despierres.pro>
```

Comment:
Move this to the class instead, [file header is pre-defined](https://github.com/renanbr/bibtex-parser/blob/master/.php_cs)